adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureParser.scala

```diff
@@ -177,6 +185,7 @@ class IntervalListParser extends Serializable {
             Feature.newBuilder()
               .setContig(Contig.newBuilder().setContigName(fields(0)).build())
               .setStart(fields(1).toLong - 1)
+              // todo: IntervalList ranges are closed, is this correct?
```

Comment:
This is correct. Drop `todo` but keep comment as documentation? E.g., `// NOTE: IntervalList ranges are closed`